src/bptl/work_units/zgw/tasks/rol.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+from zgw_consumers.constants import APITypes
+
+from bptl.tasks.base import check_variable
+from bptl.tasks.registry import register
+
+from .base import ZGWWorkUnit
+
+logger = logging.getLogger(__name__)
+
+
+@register
+class CreateRolTask(ZGWWorkUnit):
+    """
+    Create a new ROL for the ZAAK in the process.
+
+    **Required process variables**
+
+    * ``zaakUrl``: full URL of the ZAAK to create a new rol for
+    * ``omschrijving``: roltype.omschrijving for the ROL
+    * ``betrokkeneType``: type of the betrokkene
+    * ``roltoelichting``: description of the ROL
+    * ``betrokkene``: JSON object containing the data for ROL.betrokkeneIdentificatie. See
```

Comment:
This is not what was specified in the ticket:

> `betrokkene`: a JSON object containing the rest of the body for rol_create

In this form, it's not possible to pass a URL for a `betrokkene`, as you are now enforcing everyone to use `betrokkeneIdentificatie` which should only be an escape hatch and not the de-facto way.

This is also not what happens in `bptl.work_units.zgw.tasks.zaak.CreateZaakTask.create_rol` as mentioned in the ticket.

`betrokkene` is _most_ of the body of `rol_create` operation, and by doing that, you don't need the variables `betrokkeneType` and `roltoelichting` either, since they are part of that body.

Reply:
The implementation is changed - now all required rol variables (except `zaak` and `roltype`) are retrieved from `betrokkene` input variable